web/styles/settings.css

```diff
@@ -1626,6 +1626,7 @@ $option_title_width: 180px;
 
 .settings-field-label {
     margin-bottom: var(--margin-bottom-field-description);
+    width: fit-content;
```

Comment:
Probably this should get a comment  like `/* Avoid having the clickable area extend to the full width of the containing element */`